Gopkg.lock

```diff
@@ -150,6 +150,14 @@
   revision = "b9bbc5664f49b6deec52393bd68f39830687a347"
```

Comment:
Off topic, but can we get this repository switched over to using go modules?

Reply:
I would very much like to do that as well, but I figured I would take stab at it in a separate concise dependency-swap PR.